### ISSUE TITLE
usShell throws exception when built with US_BUILD_SHARED_LIBS=OFF

### ISSUE THREAD

**Issue by jeffdiclemente** [CppMicroServices/CppMicroServices#61]
To reproduce:
* Build with -D BUILD_SHARED_LIBS:BOOL=OFF
* run usShell

This was seen on Windows and Linux. The issue should affect Mac as well (but I did not confirm).

On Linux you should see the following in a terminal:

```
terminate called after throwing an instance of 'std::runtime_error'
  what():  bundle.name is not defined in the bundle manifest.
Abort
```

When the framework starts, the bundle name for CppMicroServices can't be found, causing the exception.

**Comment by saschazelzer** [comment]
The usShell needs some work in I suspected that it is currently broken. Thanks for the report.

**Comment by saschazelzer** [comment]
This has been fixed in 68b9778.
